eng/pipelines/templates/steps/vcpkg.yml

```diff
@@ -1,146 +1,52 @@
-# Assumes that the VM image is in a variable called OSVmImage
-
 parameters:
   # Use the variable name itself (no $() or other wrapping syntax). This is
   # because we use runtime and macro expressions for conditions and script
   # invocations
   DependenciesVariableName: vcpkg.deps
 
-  # Ref at which to check out (can be commit SHA or tag in the form of
-  # `tags/<tag>`)
-  VcpkgVersion: tags/2020.06
-
 steps:
-  # Set VCPKG_INSTALLATION_ROOT location for cache on Mac OS hosts
-  - task: Bash@3
-    inputs:
-      targetType: inline
-      workingDirectory: $(Agent.TempDirectory)
-      script: |
-        echo "##vso[task.prependpath]$(Agent.TempDirectory)/vcpkg"
-        echo "##vso[task.setvariable variable=VCPKG_INSTALLATION_ROOT;]$(Agent.TempDirectory)/vcpkg"
-    displayName: MacOS - Set VCPKG_INSTALLATION_ROOT
+  - pwsh: |
+      Write-Host "##vso[task.prependpath]$(Agent.TempDirectory)/vcpkg"
+      Write-Host "##vso[task.setvariable variable=VCPKG_INSTALLATION_ROOT;]$(Agent.TempDirectory)/vcpkg"
```

Comment:
What is the `;` in these set variables statements for?

Reply:
Instead of using the temp directory it might be worth putting this into a directory in the repo that we .gitignore so it is a little closer to the usage and less error prone then being in the temp directory.

Reply:
The `;` is left over from a copy/paste and is redundant. It separates the variable name from other parameters like `issecret`